### PR TITLE
DHFPROD-4390: Create merge rule form

### PR DESCRIPTION
**Pull request by xnikhil08**
### Description
Creating a dummy structure for merge rule dialog with property selector mocked to Customer Entity. Only one field "Property" is available in this ticket. Other properties are being added as part of separate ticket.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    const displayAddMergeRuleDialog = () => {
+        setActiveStep([], props.entityModel['model']['definitions'], props.entityName);
+        toggleMergeRuleDialog(true);
+        setOpenAddMergeRuleDialog(true);
+    }
+    /*---------------------------------------*/
+    return (
+        <div className={styles.mergingContainer}>
+            {/* Below button and the ternary operation based on showMergeRuleDialog should be removed/modified when working on CRUD merge step */}
+            <MLButton type="primary" onClick={displayAddMergeRuleDialog}>Add (Temp button to test Rule dialog)</MLButton>
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx:31]: This button(and related code) is purely temporary and is created just so it's easier to verify/initiate the merge rule dialog, until the CRUD Merge Step story is finished.

Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx:31]: This button(and related code) is purely temporary and is created as a placeholder, just so it's easier to verify/initiate the merge rule dialog, until the CRUD Merge Step story is finished.

Comment (xnikhil08) [inline marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx:31]: This button(and related code) is purely temporary and is created as a placeholder, just so it's easier to verify/initiate the merge rule dialog, until the CRUD Merge Step story is finished.

**Review by brucean52 — APPROVED** [review]

**Review by Sanjeevani19 — COMMENTED** [review]
@xnikhil08 Reviewed, Looks good. Could you please add the tests you added to the test plan document (https://wiki.marklogic.com/pages/viewpage.action?pageId=175487374)

File: marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx (hunk 1 above)
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/components/entities/merging/merging-card.tsx:31]: > @xnikhil08 Reviewed, Looks good. Could you please add the tests you added to the test plan document.(https://wiki.marklogic.com/pages/viewpage.action?pageId=175487374)

@Sanjeevani19 - Done.

**Review by Sanjeevani19 — APPROVED** [review]

**Pull request merged**